lib/streamlit/elements/widgets/time_widgets.py

```diff
@@ -66,6 +66,7 @@
 DateWidgetReturn: TypeAlias = Union[
     date, Tuple[()], Tuple[date], Tuple[date, date], None
 ]
+DateTupleReturn: TypeAlias = Union[Tuple[()], Tuple[date], Tuple[date, date], None]
```

Comment:
What do you think about calling `DateTupleReturn` something like `DateWidgetRangeReturn` to help document that this is used to specify date ranges?

Reply:
Yeah, Sounds good to me. It would certainly help to clearly document that this is used specifically to specify date ranges and helps with readability.